### PR TITLE
Add more fields to PyPI metadata

### PR DESCRIPTION
**Pull request by jl-wynen**
Just a few things that I noticed where missing.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -61,4 +66,20 @@ def get_cmake_args():
               'jupyterlab-widgets', 'jupyter_nbextensions_configurator', 'nodejs',
               'pythreejs'
           ],
-      })
+      },
+      classifiers=[
+          'Development Status :: 3 - Alpha',
```
Comment (SimonHeybrock) [inline setup.py:71]: Is it?

File: setup.py [hunk 2]
```diff
@@ -61,4 +66,20 @@ def get_cmake_args():
...
+          'Programming Language :: Python :: 3',
+          'Programming Language :: Python :: 3 :: Only',
+          'Programming Language :: Python :: 3.8',
+          'Programming Language :: Python :: 3.9',
+          'Programming Language :: Python :: 3.10',
+          'Operating System :: MacOS :: MacOS X',
+          'Operating System :: Microsoft :: Windows',
+          'Operating System :: POSIX :: Linux',
+          'Topic :: Scientific/Engineering',
+          'Typing :: Typed',
```
Comment (SimonHeybrock) [inline setup.py:84]: What is this?

File: setup.py (hunk 1 above)
Comment (jl-wynen) [reply setup.py:71]: What would you put instead? It's not production ready. And for a beta, we would have to have all features in place and only make smaller adjustments and fixes.

File: setup.py (hunk 2 above)
Comment (jl-wynen) [reply setup.py:84]: The package has type annotations.

**Comment by jl-wynen** [comment]
Removed the 'Development Status' classifier. Is there anything else I need to change?

**Review by SimonHeybrock — APPROVED** [review]

**Pull request merged**